mithril-common/src/era/supported_era.rs

```diff
@@ -0,0 +1,16 @@
+/// The era that the software is running or will run
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum SupportedEra {
+    /// Thales era
+    Thales,
+    /// Pythagoras era
+    Pythagoras,
```

Comment:
We should remove the second Era from our first design since it was only there for test purpose :).

Reply:
Done :heavy_check_mark: